OWNERS

```diff
@@ -0,0 +1,47 @@
+# Core maintainers
+# GitHub org: rootless-containers/slirp4netns-approvers
+approvers:
+  - AkihiroSuda
+  - giuseppe
+  - cyphar
+
+# ----------
+# Package maintainers
+# GitHub Team: rootless-containers/slirp4netns-package-maintainers
+
+# https://src.fedoraproject.org/rpms/slirp4netns
+# https://copr.fedorainfracloud.org/coprs/vbatts/shadow-utils-newxidmap/package/slirp4netns/
+fedora-package-maintainers:
+  - lsm5
+  - rhatdan
+  - giuseppe
+  - vbatts
+
+# https://www.archlinux.org/packages/community/x86_64/slirp4netns/
+arch-package-maintainers:
+  - barthalion
+
+# https://build.opensuse.org/package/show/openSUSE%3AFactory/slirp4netns
+opensuse-package-maintainers:
+  - saschagrunert
+
+# https://packages.debian.org/buster/slirp4netns
+debian-package-maintainers:
+  - siretart
```

Comment:
I'd prefer to be mentioned with full name and email address. Would that be possible?

Reply:
I just followed Kubernetes `OWNERS` syntax.  Alternatively we may use [Moby-style](https://github.com/moby/moby/blob/master/MAINTAINERS) or other syntax, but wondering other 
 package maintainers might not want to list email address.

Reply:
From a packaging perspective, it is imperative for complying to Debian policy to clearly identify the copyright and their copyright holders. This usually includes mentioning a way to contact the original authors, and including the email is the most straight-forward way. I'm sure most mainstream Linux distributions have similar policies around.

I think the `OWNERS` file would be a great opportunity to make the life of package maintainers easier. I'd appreciate a quick review of https://sources.debian.org/src/slirp4netns/0.2.1-1/debian/copyright/ and point out inaccuracies or omissions.  If an `OWNERS` file had been around at the time I initially packaged this software, it would have been my primary source of information for the `debian/copyright` file.

Reply:
@siretart [Moby-style](https://github.com/moby/moby/blob/master/MAINTAINERS) `MAINTAINERS` file works for you?

For email addresses, is it ok to include only core maintainers addresses?

@giuseppe @cyphar let me know your email addresses you prefer to use here.

Reply:
`cyphar@cyphar.com`.

Reply:
`Giuseppe Scrivano <giuseppe@scrivano.org.>`

Reply:
Thanks, updated PR to use Moby-style TOML syntax with email addresses of Approvers.

Reply:
The moby example looks interesting, thanks for looking into that.
I don't have a strong opinion or preference on the formatting. The documentation of copyright and copyright holders is more important IMO.

Reply:
IANAL, but perhaps `debian/copyright` should be written more consistent with https://sources.debian.org/src/qemu/1:3.1+dfsg-4/debian/copyright/ ?

W.r.t. slirp, `qemu/debian/copyright` just says `The slirp code is released under the following BSD license: (3-clause BSDL text)` but it contains more lines for copyright holders. 

(What is "copyright holders" anyway? "copyright holders" != "contributors"?)

Reply:
cc @elmarco @sthibaul for slirp license clarification

Reply:
You don't need to be a lawyer. If you are asking about the requirements of `debian/copyright`, the requirements are spelled out at https://www.debian.org/doc/debian-policy/ch-docs.html#copyright-information - I'm following section 12.5.1 (machine-readable `debian/copyright`) which at the option of the package maintainer. The qemu package apparently chose not to use that.

Copyright holders are individuals or companies that hold one or more of the exclusive rights of copyright in the work (cf. https://www.csusa.org/page/Definitions#copyrightholder). To give a concrete example, I would myself consider a contributor, because I'm participating in this discussion, bugs, and packging for Debian. But I would not claim any rights on the work itself. Does this make sense?

Reply:
Thanks, slirp4netns debian/copyright LGTM.

@giuseppe LGTY?